ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Node.hs

```diff
@@ -567,9 +567,11 @@ registerInitialFunds initialFunds nes = nes {
           SL._reserves = reserves <-> coin (SL.balance initialFundsUtxo)
         }
 
+    initIncrStake = SL._stakeDistro utxoState
     ledgerState' = ledgerState {
           SL._utxoState = utxoState {
-              SL._utxo = utxo'
+              SL._utxo = utxo',
+              SL._stakeDistro = SL.updateStakeDistribution initIncrStake (SL.UTxO mempty) utxo'
```

Comment:
Would you add a note (or use a local binding) that makes it clear the `SL.UTxO mempty` is the set of UTxO to delete?